addons/point_of_sale/static/src/js/Misc/AbstractReceiptScreen.js

```diff
@@ -44,8 +45,12 @@ odoo.define('point_of_sale.AbstractReceiptScreen', function (require) {
          */
         async _printWeb() {
             try {
-                const isPrinted = document.execCommand('print', false, null);
-                if (!isPrinted) window.print();
+                if (methods.switchAccount) {
+                    window.print();
+                } else {
+                    const isPrinted = document.execCommand('print', false, null);
```

Comment:
@caburj  Do you remember in which cases window.print doesn't work? I already check the link but I also see that we only do 'document.execCommand('print', false, null);' for Safari in Odoo see https://github.com/odoo/odoo/blob/4629a4fb0eacedb024562ef5542011d2e0b2c1f4/addons/point_of_sale/static/src/js/screens.js#L1490.

Reply:
In the stackoverflow link, I think I accepted the answer without any proper investigation. Therefore, I don't remember anything about which browser/version the `window.print` doesn't work. Basically, I assumed that for all, `document.execCommand('print', false, null)` should be called first before `window.print`, but based on this PR, it turns out that result of `document.execCommand('print', false, null)` is not reliable on mobile.

I checked this: https://caniuse.com/?search=window.print and it appears that window.print seems to be safe.